src/com/ms/silverking/cloud/dht/daemon/MessageModule.java

```diff
@@ -100,7 +101,7 @@
     private final StorageProtocol localConsistencyModeToStorageProtocol[];
     private final RetrievalProtocol consistencyModeToRetrievalProtocol[];
     
-    private static final boolean    debug = false;
+    private static final boolean    debug = PropertiesHelper.systemHelper.getBoolean("com.ms.silverking.cloud.dht.daemon.MessageModule.debug", false);
```

Comment:
Should use a property name for this instead of the hardcode. Something like:
private static final String    debugProperty = MessageModule.class.getCanonicalName() +".Debug";